charts/victoria-metrics-k8s-stack/templates/rules/k8s.rules.container_cpu_usage_seconds_total.yaml

```diff
@@ -3,7 +3,7 @@ Generated from 'k8s.rules.container_cpu_usage_seconds_total' group from https://
 Do not change in-place! In order to change this file first read following link:
 https://github.com/VictoriaMetrics/helm-charts/tree/master/charts/victoria-metrics-k8s-stack/hack
 */ -}}
-{{- if and .Values.defaultRules.create  }}
+{{- if and .Values.defaultRules.create .Values.ContainerCpuUsageSecondsTotal.enabled .Values.defaultRules.rules.ContainerCpuUsageSecondsTotal }}
```

Comment:
is this coming from the last sync_rules.py run?

Reply:
No it didn't. fixed with your new convention